### PR TITLE
Add mark calls answered elsewhere when cancelled to groups and queues

### PR DESCRIPTION
**Pull request by bloom1**
Depends-On: https://github.com/wazo-platform/wazo-confd/pull/73

### CODE REVIEW

File: wazo_ui/plugins/group/form.py [hunk 1]
```diff
@@ -75,6 +75,7 @@ class GroupForm(BaseForm):
     schedules = FieldList(FormField(ScheduleForm), min_entries=1)
     call_permission_ids = SelectMultipleField(l_('Call Permissions'), choices=[])
     call_permissions = FieldList(FormField(CallPermissionForm))
+    mark_answered_elsewhere = BooleanField(l_('Mark all calls as answered elsewhere when cancelled'), default=False)
```
Comment (pc-m) [inline wazo_ui/plugins/group/form.py]: I think `True` would be a better default

File: wazo_ui/plugins/queue/form.py [hunk 2]
```diff
@@ -102,6 +102,7 @@ class QueueForm(BaseForm):
 
     enabled = BooleanField(l_('Enabled'), default=False)
     members = FormField(MembersForm)
+    mark_answered_elsewhere = BooleanField(l_('Mark all calls as answered elsewhere when cancelled'), default=False)
```
Comment (pc-m) [inline wazo_ui/plugins/queue/form.py]: I think `True` would be a better default

Comment (bloom1) [reply wazo_ui/plugins/queue/form.py]: It's not really a default value... It's more a value if the checkbox is not checked. So if you try to disable the feature when default=True, it always sets it to True.

**Comment by pc-m** [comment]
recheck

**Pull request merged**